__tests__/todoList.spec.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react-native';
+import { userDataBuilder, itemDataBuilder } from 'dataBuilders';
+import { submitInput, submitMultiples } from 'submitInput';
+
+import Todo from '../src/screens/todo';
+
+const user = userDataBuilder();
+const todoItem = itemDataBuilder();
+
+test('all item should be toggle at the same time', () => {
+  //* render the todo screen
+  const { getByPlaceholderText, getByText, getByLabelText, getAllByRole } =
+    render(<Todo user={user} />);
+
+  //* get to-do list, items counter and input
+  const counter = getByText(/active: */i);
+  const input = getByPlaceholderText(/What needs to be done?/i);
+
+  //* type in the input field & click submits multiple times
+  submitMultiples(input, todoItem.extraWords);
```

Comment:
Why do we need to emulate this?

Reply:
To make this tests work, its needed more than one item, a make a quick function to emulate the input of different words, its quicker that way than copy & paste `submitInput()` multiple times